music/genre.rb

```diff
@@ -0,0 +1,14 @@
+class Genre
+  attr_accessor :name, :items
+
+  def initialize(name, _item)
+    @id = Random.rand(1..1000)
+    @name = name
+    @item = []
+  end
```

Comment:
- [ ] In the Genre class, the parameter _item is not being used. you can remove this parameter.
- [ ] In the Genre class, there is no need to generate an id attribute since it is not being used, you can go ahead and remove it to make your code look more clean.👍